modules/nf-core/chromap/index/meta.yml

```diff
@@ -15,6 +15,11 @@ tools:
       licence: ["GPL v3"]
 
 input:
+  - meta:
```

Comment:
missing meta2?

Reply:
This is for the indexing module and only has one meta for the `fasta` file, see [here](https://github.com/JoseEspinosa/nf-core-modules/blob/cfdc086f72344be9051b09903af043efbdd41660/modules/nf-core/chromap/index/main.nf#L10-L17)